### ISSUE TITLE
Add all Twisted Names howto documents to the howto/index.rst toctree

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6940]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)| @wallrj reported|
|-|-|
|Trac ID|trac#6940|
|Type|task|
|Created|2014-02-01 16:45:42Z|
|Branch|https://github.com/twisted/twisted/tree/names-howto-toctree-6940|

The two new Twisted Names howto documents are missing a "Twisted Names" element in their breadcrumb navigation on readthdocs
 * http://twisted.readthedocs.org/en/latest/projects/names/howto/names.html, vs
 * http://twisted.readthedocs.org/en/latest/projects/names/howto/client-tour.html

Because I forgot to add links to those documents to the toctree in 
 * [docs/projects/names/howto/index.rst](docs/projects/names/howto/index.rst)

<details><summary>Searchable metadata</summary>

```
trac-id__6940 6940
type__task task
reporter__rwall rwall
priority__normal normal
milestone__ 
branch__branches_names_howto_toctree_6940 branches/names-howto-toctree-6940
branch_author__rwall rwall
status__closed closed
resolution__fixed fixed
component__names names
keywords__documentation documentation
time__1391273142000000 1391273142000000
changetime__1391275356000000 1391275356000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_1"></a>|@wallrj commented|
|-|-|

(In [41511]) Branching to 'names-howto-toctree-6940'

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_2"></a>|@wallrj commented|
|-|-|

Ready for review in log:branches/names-howto-toctree-6940

 * It's not easy to see the effect of this in the locally rendered docs, since they don't seem to include an index or breadcrumb trail,
 * but you should notice that two toctree warnings have disappeared when the docs are build. Eg

   ```
   cd docs
   make html
   ...
   checking consistency...
   ...
   /home/richard/projects/Twisted/branches/names-howto-toctree-6940/docs/projects/names/howto/client-tour.rst:: WARNING: document isn't included in any toctree
   /home/richard/projects/Twisted/branches/names-howto-toctree-6940/docs/projects/names/howto/custom-server.rst:: WARNING: document isn't included in any toctree
   ...
   done

   ```

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_3"></a>|@hynek commented|
|-|-|

Please add an comment to [#6905](https://github.com/twisted/twisted/issues/6905) so that this potential mistake is documented in the new style guide and merge.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_4"></a>|@wallrj set status to `closed`|
|-|-|

(In [41514]) Merge names-howto-toctree-6940

Author: rwall
Reviewer: hynek
Fixes: [#6940](https://github.com/twisted/twisted/issues/6940)

Add toctree links to twisted.names howto index.
